modules/social_features/social_event/social_event.install

```diff
@@ -1555,3 +1555,13 @@ function social_event_update_8810() {
   // Output logged messages to related channel of update execution.
   return $updateHelper->logger()->output();
 }
+
+/**
+ * Show user time zone as default.
+ */
+function social_event_update_8811(&$sandbox) {
```

Comment:
@dmytriikaiun I think this is fine to set this new option by default for new installations, but I think usually we don't force update it for all existing platforms. So I think we don't need this hook update. 
What do you think @ronaldtebrake, @bramtenhove?

Reply:
Good point @ribel 
I think you are right, we rather explain to the SM how they can do it if they want to. Allowing them to "opt-in" to this feature for existing platforms. Seems like you have taken that scenario in to account already so that is great and I can totally imagine this update hook can be removed. 
This way it can also land in 9.1 because we don't force a new feature to everybody.